src/tutorial/TutorialState.cs

```diff
@@ -152,6 +161,15 @@ public bool WantsNearbyCompoundInfo()
         return MicrobeMovement.Complete && !GlucoseCollecting.Complete && GlucoseCollecting.CanTrigger;
     }
 
+    /// <summary>
+    ///   Returns true when the tutorial system is in a state where nearby engulfable entity info is wanted
+    /// </summary>
+    /// <returns>True when the tutorial system wants engulfable entity information</returns>
+    public bool WantsNearbyEngulfableInfo()
+    {
+        return GlucoseCollecting.Complete;
```

Comment:
For performance reasons, I'd really like that this isn't permanently on, so could this stop returning true once the relevant tutorials are complete?